jwql/tests/test_api_views.py

```
@@ -90,10 +99,8 @@ def test_api_views(url):
 
     try:
         url = request.urlopen(url)
-    except error.HTTPError as e:
-        if e.code == 502:
-            pytest.skip("Server problem")
-        raise(e)
+    except (error.HTTPError, error.URLError):
```

So the HTTP Error makes sense if the server is down or we can't connect, what does the adding the URL error catch here? Do we expect to build URLs that might not exist on the app in this test?

---

I included the URL catch in case some of the specific data referenced on central storage goes missing again.  I can let that fail instead if you prefer.